app/src/main/java/com/github/gotify/NotificationSupport.java

```diff
@@ -32,19 +36,69 @@ public static void createChannels(NotificationManager notificationManager) {
                             Channel.FOREGROUND,
                             "Gotify foreground notification",
                             NotificationManager.IMPORTANCE_LOW);
-            // High importance for message notifications so that they are shown as heads-up
-            // notifications and sorted towards the top of the notification shade
-            NotificationChannel messages =
+
+            NotificationChannel messagesImportanceMin =
+                    new NotificationChannel(
+                            Channel.MESSAGES_IMPORTANCE_MIN,
+                            "Min priority messages (<1)",
+                            NotificationManager.IMPORTANCE_MIN);
+
+            NotificationChannel messagesImportanceLow =
+                    new NotificationChannel(
+                            Channel.MESSAGES_IMPORTANCE_LOW,
+                            "Low priority messages (1-3)",
+                            NotificationManager.IMPORTANCE_LOW);
+
+            NotificationChannel messagesImportanceDefault =
                     new NotificationChannel(
-                            Channel.MESSAGES,
-                            "Gotify messages",
+                            Channel.MESSAGES_IMPORTANCE_DEFAULT,
+                            "Normal priority messages (4-7)",
+                            NotificationManager.IMPORTANCE_DEFAULT);
+            messagesImportanceDefault.enableLights(true);
+            messagesImportanceDefault.setLightColor(Color.CYAN);
+
+            NotificationChannel messagesImportanceHigh =
+                    new NotificationChannel(
+                            Channel.MESSAGES_IMPORTANCE_HIGH,
+                            "High priority messages (>7)",
                             NotificationManager.IMPORTANCE_HIGH);
-            messages.enableLights(true);
-            messages.setLightColor(Color.CYAN);
+            messagesImportanceHigh.enableLights(true);
```

Comment:
Currently it only does a sound but it is not vibrating anymore maybe we could add android.app.NotificationChannel#enableVibration to both high and normal priority?

Reply:
Ah yes, you're right. Totally missed that vibration is disabled by default.